test/cmake/compress-and-verify.cmake

```diff
@@ -137,24 +172,27 @@ message(STATUS "Decompress ${DECOMPRESS_COMMAND}")
 message(STATUS "  Input: ${OUTPUT_BASE}.gz")
 message(STATUS "  Output: ${OUTPUT_BASE}")
 
-execute_process(COMMAND ${CMAKE_COMMAND}
-    "-DCOMMAND=${DECOMPRESS_COMMAND}"
-    -DINPUT=${OUTPUT_BASE}.gz
-    -DOUTPUT=${OUTPUT_BASE}
-    "-DSUCCESS_EXIT=${SUCCESS_EXIT}"
-    -P ${CMAKE_CURRENT_LIST_DIR}/run-and-redirect.cmake
-    RESULT_VARIABLE CMD_RESULT)
+if(FILEMODE)
+    exec_files("${DECOMPRESS_COMMAND}" "${OUTPUT_BASE}.gz")
+else()
+    exec_streams("${DECOMPRESS_COMMAND}" "${OUTPUT_BASE}.gz" "${OUTPUT_BASE}")
+endif()
 
 if(CMD_RESULT)
     cleanup()
     message(FATAL_ERROR "Decompress failed: ${CMD_RESULT}")
 endif()
 
 if(COMPARE)
+    message(STATUS "xxd comparison")
```

Comment:
What's xxd mean?

Reply:
`xxd` is the command you use in the diff macro..
It might be misleading though, since it is only used if there is a difference.